### ISSUE TITLE
Screen Reader: About: Keyboard shortcuts are announced without additional text description

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77455]
## Action Performed:
Precondition: The User should be logged in

1. Using Android TalkBack or iOS VoiceOver, open the Expensify app
2. Swipe to the 'Account' button at the bottom right corner of the screen and double-tap to activate
3. Swipe to the 'About' button and double-tap to activate
4. Swipe to the 'Keyboard shortcuts' button and double-tap to activate
5. Swipe between shortcut options
6. Note how the controls are announced.

## Expected Result:
The expected result is a complete announcement that includes the supplementary shortcut information: "Shift + Esc, mark all messages as read."

## Actual Result:
The text blocks are announced without part the displayed texts.

## Area issue was found in:
Preferences and  Subsections

## Failed WCAG checkpoints
WCAG 2.4.6 Headings and Labels
https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels

## User impact:
People with mobility impairments who must rely on keyboard access for operating a page benefit from a logical, usable focus order. People with vision impairments or disabilities that make reading difficult can become disoriented when tabbing takes focus someplace unexpected or when they cannot easily find the content surrounding an interactive element. Only a small portion of the page may be visible to an individual using a screen magnifier at a high level of magnification. Such a user may interpret a field in the wrong context if the focus order is not logical.

## Suggested resolution:
Ensure that all the displayed information of the shortcut is included in the accessibilityLabel.
https://reactnative.dev/docs/accessibility#accessibilitylabel

## Environment:
Samsung Galaxy S24 Ultra - Android 14, Partner - Native (No mobile browser)
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos

https://github.com/user-attachments/assets/f401b6c4-9668-4fc3-94f1-a63c9f3d6873

![Image](https://github.com/user-attachments/assets/1089f3c7-62f6-44cc-8760-661e39dd82a7)

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7006473
**Issue reported by:** NA
**Slack conversation:** NA
